src/docs/slot_specifications.md

```diff
@@ -0,0 +1,299 @@
+
+### 5.2 Title format
+
+The item (title) SHOULD be lower case, including first character of the item.
+
+- Valid example: `library size`.
+- Invalid examples:
+  - `Library size` (capitalisation of first character).
+  - `Library Size` (capitalisation of of all words).
+
+Capitalisation MAY be used when it is an acronym or abbreviation that typically used capitalisation in the English language (e.g. `DNA`, `API`, `pH`).
+
+- Valid example: `MAG coverage software`.
+- Valid example: `API gravity`.
+
+## 6. Term examples attribute
+
+<!-- JFY comment: this is a new guideline I would like to propose, so requires discussion  -->
+
+### 6.1 Minimum number of examples
+
+There MUST have minimum of 3 examples for a term (slot).
+
+### 6.2 Scope of examples
+
+In addition to the minimum number of examples, the examples SHOULD cover the full range of possible values, string formats, or any other way that information can be given to the term (slot).
+
+For example if a term (slot) accepts either an ontology term _or_ a free text string, there should be at least one example for each type.
+If a term (slot) accepts different unit types, there should be at least two examples of different units to demonstrate multiple units are accepted.
+
+### 6.3 Examples for terms that allow more than one entry
+
+If a term (slot) allows multiple occurrences ('multivalued'), the examples MUST include at a minimum two examples, one to show inputting a single value, and another to show how to fill the term with multiple values.
+
+## 7. Term section attribute
+
+> [!WARNING]
+> The guidance in this section regarding `subset`s may be replaced with the use of `slot-group` in the future.
+
+### 7.1 All core slots must be assigned a subset
+
+All core checklist LinkML slots (terms) MUST be assigned to a section (subset).
+
+### 7.2 All extension terms must not be assigned a subset
+
+A slot (term) assigned to just an extension MUST NOT be assigned to a section (subset).
+
+## 8. Term keywords attribute
+
+### 8.1 Number of keywords
+
+All term (slots) MUST have at least one keyword.
+
+### 8.3 Keywords should be re-used
+
+Re-using existing keywords SHOULD be preferred, but new keywords MAY be created if needed.
+
+### 8.2 Keyword types
+
+Keywords SHOULD be descriptive of the data the term is intended to hold in a way it can be grouped with with other terms (slots).
+
+This can correspond to stage of project, domain of research, or the sample type (or extension) the term is intended to be used with.
+
+It MAY ALSO include each descriptive part of the title (item) in full words (e.g. `air_temp` could have keywords `air` and `temperature`).
+
+## 9. Term MIxS ID attribute
+
+### 9.1 MIxS ID requirement
+
+The term MUST have a MIxS ID (slot_uri) that is unique within the MIxS model.
+
+### 9.2 MIxS ID format
+
+The MIxS ID (slot_uri) must begin with the string `MIXS`, a colon, and followed by a 7 digit number.
+
```

Comment:
is it worth saying that these will be assigned by GSC-CIG so users/submitters dont need to think about this during development?